### PR TITLE
fix: rule order-properties not work

### PR DESCRIPTION
**Pull request by hyoban**
## PR Checklist

-   [ ] Addresses an existing open issue: fixes #000
-   [ ] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
-   [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Just a quick fix. I don't know why this worked before.

### CODE REVIEW

File: src/rules/order-properties.ts
Comment (JoshuaKGoldberg) [inline src/rules/order-properties.ts:1]: [Testing] Oh, nice catch - this only happens if there are multiple failures, right? Can you add a unit test please?

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
oopity

File: src/rules/order-properties.ts
Comment (hyoban) [reply src/rules/order-properties.ts:1]: Added, just make sure the first key is corrected.

**Review by JoshuaKGoldberg — APPROVED** [review]
Swell, thanks!

**Pull request merged**